core/app/assets/js/ui/modal/modal.js

```diff
@@ -42,9 +42,14 @@ ModalProvider.propTypes = {
 	children: PropTypes.node.isRequired,
 	toggleButtonProps: PropTypes.object.isRequired,
 	title: PropTypes.string.isRequired,
+	icon: PropTypes.string,
 };
 
-class Modal extends React.Component {
+ModalProvider.defaultProps = {
```

Comment:
Remove and use this icon from where the ModalProvider is rendered.

Reply:
Handled.